src/components/form/form/form.tsx

```diff
@@ -205,11 +217,16 @@ export const Form: React.FC<FormProps> = ({
               getValueFromFormData(fields, valuesState, field);
 
             const error = getErrorFromErrors(fields, errorsState, field);
+            const message =
+              error === 'Veld "{name}" is verplicht'
```

Comment:
Can we move this to a constants somwhere next to the validator? And us it a much as possible (except for the `useInt.formatMessage` call which needs to be a string literal)?